docker-compose.yml

```diff
@@ -16,6 +16,8 @@ services:
       - elasticsearch
       - redis
       - postgres
+    volumes:
+      - "./config/config.json:/home/bookbrainz/bookbrainz-site/config/config.json:ro"
```

Comment:
I see this going away when the entire directory is mounted with automatic reloading of sites. So looks good to me for now.